### DISCUSSION TITLE
How to Link Git Repo to Vercel of An Organization of Which you are a member

### DISCUSSION THREAD

**Discussion by NithikYekollu** [vercel/community#7105] [Help] [answered]
### Summary

I was trying to deploy my project in vercel, but since moving it to an org on github that I am a member of,  I was unable to add the github link to vercel. It was saying only org owners were able to do that. Is there a way to do this without needing me to be the admin of the whole org?

### Example

NA

### Steps to Reproduce

N/A

**Comment by versecafe** [comment, marked as answer]
@NithikYekollu If you have admin privileges for the repository you might be able to I don't remember, but you can also just ask for approval for the Vercel extension from your orgs admin

[a comment by an automated account is not shown]
